resolwe/storage/management/commands/run_storage_cleanup.py

```diff
@@ -0,0 +1,39 @@
+""".. Ignore pydocstyle D400.
+
+============================
+Command: run_storage_cleanup
+============================
+
+"""
+import logging
+
+from asgiref.sync import async_to_sync
+from channels.layers import get_channel_layer, ChannelFull
+
+from django.core.management.base import BaseCommand
+
+from resolwe.storage.protocol import (
+    CHANNEL_STORAGE_CLEANUP_WORKER,
+    TYPE_STORAGE_CLEANUP_RUN,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
Here processing one filestorage object at a time would come handy, since I could notify cleanup manager to only process filestorage object belonging to the deleted data object. I will rewrite the code.